.github/workflows/system-tests-opensource.yml

```diff
@@ -164,47 +164,30 @@ jobs:
       run: |
         # TODO: There are a couple of modifications to the helm chart that we are doing right now:
         #       1. The grafana prometheus stack is disabled as there are currently no system tests checking its
         #          functionality. Once the model monitoring feature is complete and we have system tests for it, we
-        #          can enable it. (flags: --set kube-prometheus-stack.enabled=false)
+        #          can enable it.
         #       2. The mlrun DB is set as the old SQLite db. There is a bug in github workers when trying to run a mysql
         #          server pod in minikube installed on the worker, the mysql pod crashes. There isn't much information
         #          about this issue online as this isn't how github expect you to use mysql in workflows - the worker
         #          has a mysql server installed directly on it and should be enabled and used as the DB. So we might
         #          want in the future to use that instead, unless the mysql will be able to come up without crashing.
-        #          (flags: --set mlrun.httpDB.dbType="sqlite" --set mlrun.httpDB.dirPath="/mlrun/db"
-        #          --set mlrun.httpDB.dsn="sqlite:////mlrun/db/mlrun.db?check_same_thread=false"
-        #          --set mlrun.httpDB.oldDsn="")
-        helm --namespace ${NAMESPACE} \
-            install mlrun-ce \
-            --debug \
-            --wait \
-            --timeout 600s \
-            --set kube-prometheus-stack.enabled=false \
-            --set mlrun.httpDB.dbType="sqlite" \
-            --set mlrun.httpDB.dirPath="/mlrun/db" \
-            --set mlrun.httpDB.dsn="sqlite:////mlrun/db/mlrun.db?check_same_thread=false" \
-            --set mlrun.httpDB.oldDsn="" \
-            --set global.registry.url=$(minikube ip):5000 \
-            --set global.registry.secretName="" \
-            --set global.externalHostAddress=$(minikube ip) \
-            --set nuclio.dashboard.externalIPAddresses[0]=$(minikube ip) \
-            --set mlrun.api.image.repository=${{ steps.computed_params.outputs.mlrun_docker_registry }}${{ steps.computed_params.outputs.mlrun_docker_repo }}/mlrun-api \
-            --set mlrun.api.image.tag=${{ steps.computed_params.outputs.mlrun_docker_tag }} \
-            --set mlrun.ui.image.repository=ghcr.io/mlrun/mlrun-ui \
-            --set mlrun.ui.image.tag=${{ steps.computed_params.outputs.mlrun_ui_version }} \
-            --set mlrun.api.extraEnvKeyValue.MLRUN_HTTPDB__BUILDER__MLRUN_VERSION_SPECIFIER="mlrun[complete] @ git+https://github.com/mlrun/mlrun@${{ steps.computed_params.outputs.mlrun_hash }}" \
-            --set mlrun.api.extraEnvKeyValue.MLRUN_IMAGES_REGISTRY="${{ steps.computed_params.outputs.mlrun_docker_registry }}" \
-            mlrun-ce/mlrun-ce
+        python automation/deployment/ce.py deploy \
+            --verbose \
+            --minikube \
+            --namespace=${NAMESPACE} \
+            --registry-url=$(minikube ip):5000 \
```

Comment:
i think that using `--minikube` and not providing `registry-url` can be automatically done by script (meaning, autocomplete the address if `minikube` is set